app/controllers/admin/project_groups_controller.rb

```diff
@@ -54,6 +54,7 @@ def destroy
 
   def render_modal_partial(status: 200)
     link_params = params.slice(:project_id, :parent_id)
+    @parents = @project.timeline_groups_preordered.to_a - [@entry] - @entry.descendants.to_a
```

Comment:
I was unable to get `@entry.self_and_descendants` working. It either is empty when used as `[@entry.self_and_descendants]` or results in an error when used as `@entry.self_and_descendants.to_a`.